config/detekt/detekt.yml

```
@@ -0,0 +1,1069 @@
+    active: true
+  UtilityClassWithPublicConstructor:
+    active: true
+  VarCouldBeVal:
+    active: true
+    ignoreLateinitVar: false
+  WildcardImport:
+    active: true
+    excludeImports:
+      - 'java.util.*'
+
+formatting:
```

YAMLlint (1.37.1)

[error] duplication of key "formatting" in mapping

(key-duplicates)